config/default.yml

```diff
@@ -113,6 +113,13 @@ Performance/Detect:
   VersionAdded: '0.30'
   VersionChanged: '1.8'
 
+Performance/DeterministicRegexp:
+  Description: 'This cop identifies places where `split` argument can be replaced from a deterministic regexp to a string.'
+  SafeAutoCorrect: true
```

Comment:
This setting is not required because `SafeAutoCorrect: true` by default. Can you remove it?